# ONEIDA - Genomics Training Course 
From reads to results

 
## Dates: 2-4 May 2018
 
_Organisers:_ 
<br>Mário Ramirez            IMM/FMUL
<br>João André Carriço        IMM/FMUL
<br>Pedro Fernandes         IGC
<br>Daniel Sobral            IGC

_Instructors:_
<br>Belén Carbonetto        IGC
<br>João André Carriço        IMM/FMUL
<br>Miguel Machado        IMM/FMUL
<br>Bruno Ribeiro Gonçalves    IMM/FMUL
<br>Mário Ramirez            IMM/FMUL
<br>Mickael Silva            IMM/FMUL
<br>Daniel Sobral            IGC
 
# Target audience:
Researchers with interest in high throughput sequence data analysis in the context of microbial pathogens, willing to analyse their own data
 
# Pre-requisites:
The Linux environment will be used in all the sessions. 
<br>Familiarity with Linux basics is required, namely basic command structure, file system navigation, redirection, piping, etc.

# Course Objectives
The objectives of the training course are to provide a hands-on tutorial to ONEIDA members of the bioinformatics software being developed and used within the scope of ONEIDA.
 
Participants will use provided datasets or their own data sets to apply the different analytical pipelines that will be presented and results will be discussed among faculty and participants. 
 
## Preliminary Schedule 
 
May 2nd              
Morning  (9:30 – 12:30)
1.     Introduction to HTS concepts
Types of studies: bacterial genome sequencing, Metagenomics (16s and shotgun), RNA-Seq

2. The raw data and available analysis: from reads to read mapping and de novo assembly  

Break (11h00-11h30)

3.     INNUCA: a genome assembly pipeline
·       
Lunch (12h30 – 13h30)

Afternoon (14h00 – 17h00)
3.     Genome annotation with Prokka
4.     Using ABRicate to identify antibiotic resistance genes.   
  Workgroup tasks
5.     Discussion
  
May 3rd 
Morning  (9:30 – 12:30)

1.     Gene-by-gene methods (chewBBACA) for Phylogenetic inference:
·      Schema creation
·      Allele calling
·      Schema evaluation

Lunch (12h30 – 13h30)

Afternoon (14h00 – 17h00)

2.     Visualizing strain relatedness results using PHYLOViZ
3.     Demonstration of the INNUENDO platform

May 4th
Morning  (9:30 – 12:30)
 
1.     Studying microbial diversity with QIIME2:
·      Pre-processing of sequences: remove primers and barcodes, demultiplex
samples
·      Quality filtering, denoising and picking of features and representative
sequences with DADA2 and Deblur
·      Assign taxonomy to features with trained classifiers.
·      Align sequences and infer phylogeny.
·      Calculate alpha and beta diversity.
·      Test for differential abundances between groups of samples using ANCOM and gneiss

Lunch (12h30 – 13h30)

Afternoon (14h00 – 17h00)

2.     Evaluating differential expression with RNA-Seq approaches
·       Aligning RNA-Seq reads to the genome using Hisat2
·       Generating Gene Counts from alignments using featureCounts
· Performing differential expression analysis using DESeq2
