# ONEIDA - Genomics Workshop
From reads to results

 
## Dates: 2-4 May 2018
 
**Organisers:**
<br>Mário Ramirez            IMM/FMUL
<br>João André Carriço        IMM/FMUL
<br>Pedro Fernandes         IGC
<br>Daniel Sobral            IGC

**Instructors:**
<br>Belén Carbonetto        IGC
<br>João André Carriço        IMM/FMUL
<br>Miguel Machado        IMM/FMUL
<br>Bruno Ribeiro Gonçalves    IMM/FMUL
<br>Mário Ramirez            IMM/FMUL
<br>Mickael Silva            IMM/FMUL
<br>Daniel Sobral            IGC
 
# Target audience:
Researchers with a strong interest in high throughput sequence data analysis in the context of microbial pathogens, that have a present or upcoming data analysis problem in their hands.
 
# Pre-requisites:
The Linux environment will be used in all the sessions. 
<br>Familiarity with Linux basics is required, at the level of basic command structure, file system navigation, redirection, piping, etc.
<br>Entry level bioinformatics knowledge will be very handy, not really required-

# Workshop Objectives
The main objectives of the workshop are:
<br>- to provide a hands-on tutorial on the software that is being developed and already in use in the scope of ONEIDA.
<br>- to provide an update on current plans in software development for ONEIDA 

# User data
Participants will be using datasets that have been prepared for demonstration purposes
<br>Participants will be allowed to bring their data sets, and test the different analytical pipelines.
<brResults will be discussed among faculty and participants. 
 
                     EM EDIÇÃO!!!!!
 
 
## Preliminary Schedule 
 
May 2nd              
Morning  (9:30 – 12:30)
1.     Introduction to HTS concepts
Types of studies: bacterial genome sequencing, Metagenomics (16s and shotgun), RNA-Seq

2. The raw data and available analysis: from reads to read mapping and de novo assembly  

Break (11h00-11h30)

3.     INNUCA: a genome assembly pipeline
·       
Lunch (12h30 – 13h30)

Afternoon (14h00 – 17h00)
3.     Genome annotation with Prokka
4.     Using ABRicate to identify antibiotic resistance genes.   
  Workgroup tasks
5.     Discussion
  
May 3rd 
Morning  (9:30 – 12:30)

1.     Gene-by-gene methods (chewBBACA) for Phylogenetic inference:
·      Schema creation
·      Allele calling
·      Schema evaluation

Lunch (12h30 – 13h30)

Afternoon (14h00 – 17h00)

2.     Visualizing strain relatedness results using PHYLOViZ
3.     Demonstration of the INNUENDO platform

May 4th
Morning  (9:30 – 12:30)
 
1.     Studying microbial diversity with QIIME2:
·      Pre-processing of sequences: remove primers and barcodes, demultiplex
samples
·      Quality filtering, denoising and picking of features and representative
sequences with DADA2 and Deblur
·      Assign taxonomy to features with trained classifiers.
·      Align sequences and infer phylogeny.
·      Calculate alpha and beta diversity.
·      Test for differential abundances between groups of samples using ANCOM and gneiss

Lunch (12h30 – 13h30)

Afternoon (14h00 – 17h00)

2.     Evaluating differential expression with RNA-Seq approaches
·       Aligning RNA-Seq reads to the genome using Hisat2
·       Generating Gene Counts from alignments using featureCounts
· Performing differential expression analysis using DESeq2
