# ONEIDA - Genomics Workshop 
for members of the team ITQB/IGC/IMM

From reads to results

 
## Dates: 2-4 May 2018
 
**Organisers:**
<br>Mário Ramirez            IMM/FMUL
<br>João André Carriço        IMM/FMUL
<br>Pedro Fernandes         IGC
<br>Daniel Sobral            IGC

**Instructors:**
<br>Belén Carbonetto        IGC
<br>João André Carriço        IMM/FMUL
<br>Miguel Machado        IMM/FMUL
<br>Bruno Ribeiro Gonçalves    IMM/FMUL
<br>Mário Ramirez            IMM/FMUL
<br>Mickael Silva            IMM/FMUL
<br>Daniel Sobral            IGC
 
# Target audience:
Researchers with a strong interest in high throughput sequence data analysis in the context of microbial pathogens, that have a present or upcoming data analysis problem in their hands.
 
# Pre-requisites:
The Linux environment will be used in all the sessions. 
<br>Familiarity with Linux basics is required, at the level of basic command structure, file system navigation, redirection, piping, etc.
<br>Entry level bioinformatics knowledge will be very handy, not really required-

# Workshop Objectives
The main objectives of the workshop are:
<br>- to provide a hands-on tutorial on the software that is being developed and already in use in the scope of ONEIDA.
<br>- to provide an update on current plans in software development for ONEIDA 

# User data
Participants will be using datasets that have been prepared for demonstration purposes
<br>Participants will be allowed to bring their data sets, and test the different analytical pipelines.
<br>Interesting results will be discussed amongst the workshop participants. 
 
## Preliminary Schedule 
 
May 2nd              

9:30 **Introduction to HTS concepts**
<br>Types of studies: bacterial genome sequencing, Metagenomics (16s and shotgun), RNA-Seq
<br>The raw data and available analysis: from reads to read mapping and de novo assembly  

11h00 Coffee Break 

11h30 **INNUCA: a genome assembly pipeline**     

12h30  Lunch @IGC Cafeteria

14h00 **Genome annotation with Prokka**

16h00 Tea Break

16h30 **Using ABRicate to identify antibiotic resistance genes**  
<br>Group work around specific 

17h30 **Discussion**
  
May 3rd

9h30 **Gene-by-gene methods (chewBBACA) for Phylogenetic inference:**
<br>- Schema creation
<br>- Allele calling

11h00 Coffee Break

11h30 **Gene-by-gene methods (chewBBACA) for Phylogenetic inference:**
<br>- Schema evaluation

12h30 Lunch @IGC Cafeteria

14h00 **Visualizing strain relatedness results using PHYLOViZ**
<br>Demonstration of the INNUENDO platform

May 4th

9h30 **Studying microbial diversity with QIIME2  (I):**
<br>- Pre-processing of sequences: remove primers and barcodes, demultiplex samples
<br>- Quality filtering, denoising and picking of features and representative sequences with DADA2 and Deblur
<br>- Assign taxonomy to features with trained classifiers.

11h00 Coffee Break

9h30 **Studying microbial diversity with QIIME2 (II):**
<br>- Align sequences and infer phylogeny.
<br>- Calculate alpha and beta diversity.
<br>- Test for differential abundances between groups of samples using ANCOM and gneiss

12h30 Lunch @IGC Cafeteria

14h00 **Evaluating differential expression with RNA-Seq approaches**
<br>- Aligning RNA-Seq reads to the genome using Hisat2
<br>- Generating Gene Counts from alignments using featureCounts
<br>- Performing differential expression analysis using DESeq2

16H00 Tea Break

16h30 **Debriefing and general discussion**

18h00 **Adjourn**

